pilot/pkg/proxy/envoy/v1/mixer.go

```diff
@@ -38,21 +40,45 @@ const (
 	// MixerFilter name and its attributes
 	MixerFilter = "mixer"
 
+	// AttrSourcePrefix all source attributes start with this prefix
+	AttrSourcePrefix = "source"
+
 	// AttrSourceIP is client source IP
 	AttrSourceIP = "source.ip"
 
 	// AttrSourceUID is platform-specific unique identifier for the client instance of the source service
 	AttrSourceUID = "source.uid"
 
+	// AttrDestinationPrefix all destination attributes start with this prefix
+	AttrDestinationPrefix = "destination"
+
 	// AttrDestinationIP is the server source IP
 	AttrDestinationIP = "destination.ip"
 
 	// AttrDestinationUID is platform-specific unique identifier for the server instance of the target service
 	AttrDestinationUID = "destination.uid"
 
+	// AttrDestinationLabels is Labels associated with the destination
+	AttrDestinationLabels = "destination.labels"
+
 	// AttrDestinationService is name of the target service
 	AttrDestinationService = "destination.service"
 
+	// AttrIP represents IP address
+	AttrIP = "ip"
+
+	// AttrUID is the uid of with source or destination.
+	AttrUID = "uid"
+
+	// AttrLabels are lables associated with source or destination.
+	AttrLabels = "labels"
+
+	// keyConfigMixer is a key in the opaque_config. It is base64(json.Marshal(ServiceConfig)).
+	keyConfigMixer = "mixer"
```

Comment:
`routeOpaqueConfigKeyMixer` and `routeOpaqueConfigKeyMixerSHA256` might be more self-describing?

Reply:
Point taken. I was referring to the name of the variable (e.g. `keyConfigMixer`) which shouldn't appear in the metadata. `mixer` as a value seems okay since it appears in the contex of the route's opaque_config.